parser.py

```diff
@@ -1 +1,40 @@
-# TODO Xin - Add methods here that we will need. Try to think carefully about organization and how to separate functionality effectively.
\ No newline at end of file
+# TODO Xin - Add methods here that we will need. Try to think carefully about organization and how to separate functionality effectively.
+from bs4 import BeautifulSoup as bs
+from bs4 import SoupStrainer as ss
+import re
+import urllib2 as ul
+ 	
+def links(webpage):
+	links = []
+	biglist = ul.urlopen(webpage)
+	soup = bs(biglist, "html.parser")
+	tablebody = soup.body.find(id="content")
```

Comment:
Why is this called tablebody?" Does the "content" id really correspond to the body of a table? Would it be better to call this "content"? I'm honestly not sure without looking at the html, which we can do together :3